pkg/store/prometheus.go

```diff
@@ -67,7 +68,9 @@ func (p *PrometheusStore) Info(ctx context.Context, r *storepb.InfoRequest) (*st
 	lset := p.externalLabels()
 
 	res := &storepb.InfoResponse{
-		Labels: make([]storepb.Label, 0, len(lset)),
+		MinTime: 0,
+		MaxTime: math.MaxInt64,
```

Comment:
I suppose this is not ideal. In the gossip we bound the MinTime by what the shipper sees as the lowest block (we do the same in the TSDBStore). But we don't have this info here. Probably not necessary for now since it's an optimization anyway.

Reply:
yeah, worth to add comment, maybe

Reply:
will do